### ISSUE TITLE
Running SPARC on AMD / ARM Environment

### ISSUE THREAD

**Issue by yasithdev** [SPARC-X/SPARC#225]
Hello, I noticed that the code requires that I use MKL to run. Is there a way to run this code on non-intel architectures? (e.g., AMD, ARM)?

**Comment by YaphetS-jx** [comment]
Yes, you can run it without MKL. In that case, you still need to have LAPACK installed on your machine, for example OpenBLAS. And in makefile, please change `USE_MKL = 0` and `USE_DP_SUBEIG = 1`. Then you should be able to compile and run SPARC.
